src/components/store/store.tsx

```diff
@@ -36,82 +33,66 @@ const storage: StateStorage = {
   },
 };
 
-// type State = MyState;
+export const createMyStore = () =>
+  createStore<
+    ExtractSliceState<typeof createIngredientsSlice> &
```

Comment:
Takhle jednoduse dostaneme ten "typ" z toho slice, at ho nemusime zbytecne exportovat a exportujeme jen to, co k necemu je.